storage/store/inmemory/unsynchronized/collections.go

```diff
@@ -0,0 +1,81 @@
+package unsynchronized
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+type Collections struct {
+	collections                    map[flow.Identifier]*flow.Collection
+	lightCollections               map[flow.Identifier]*flow.LightCollection
+	transactionIDToLightCollection map[flow.Identifier]*flow.LightCollection
+}
+
+func NewCollections() *Collections {
+	return &Collections{
+		collections:                    make(map[flow.Identifier]*flow.Collection),
+		lightCollections:               make(map[flow.Identifier]*flow.LightCollection),
+		transactionIDToLightCollection: make(map[flow.Identifier]*flow.LightCollection),
+	}
+}
+
+var _ storage.Collections = (*Collections)(nil)
+
+func (c *Collections) ByID(collID flow.Identifier) (*flow.Collection, error) {
```

Comment:
please add godocs including expected errors. it's easiest to have identical comments as what's in the interface.